packages/errors/README.md

```diff
@@ -200,6 +201,25 @@ throw new UserInputError({
 });
 ```
 
+### `BaseError`
+
+Our getting started guide talks about [Operational vs Programmer errors](../../docs/getting-started/throwing-errors.md#types-of-error). The rest of the errors in this library provide operational errors, but they all extend a base class named `BaseError`. This class offers up some of the conveniences of `OperationalError` but it's marked as non-operational:
```

Comment:
**suggestion**: in the world where BaseError existed from the start, it wouldn't make sense to talk about it "offering up some of the convenience of `OperationalError`". i think it would be clearer to new users of the library to talk about what it does in concrete terms, then mention that the other error classes inherit that.

Reply:
I did start writing it like that but I think really we don't want _loads_ of use of this? I worry slightly if all the docs for the properties are on `BaseError` then people will use that in places where `OperationalError` is more correct. It's mostly gonna be library authors who need this so I kinda wanted it to fly under the radar a little for now, alright if I make another ticket to review the docs in future and leave this as-is?

Reply:
that's fair